auth-web/src/components/auth/manage-business/AffiliationAction.vue

```diff
@@ -458,9 +467,14 @@ export default defineComponent({
         case NrRequestActionCodes.AMALGAMATE:
           goToAmalgamate(item)
           break
-        case NrRequestActionCodes.MOVE:
-          useNameRequest(item)
+        case NrRequestActionCodes.MOVE: {
+          if (isSupportedContinuationInEntities) {
+            useNameRequest(item)
+          } else {
+            goToCorpOnline()
+          }
```

Comment:
This looks different than the case for amalgamations (or others). Please follow existing code pattern (or explain why this one is different). Thanks.

Reply:
Fixed, thanks.